src/server.js

```diff
@@ -120,8 +120,8 @@ const renderImage = (params, response, next, tilePath) => {
                 )
             )
         }
-        bounds.forEach(b => {
-            if (Number.isNaN(b)) {
+        for (const b of bounds) {
```

Comment:
What is the reason for changing `forEach` to `for` loop here?  If there isn't a problem with `forEach` I'd prefer to keep as is, since it matches our looping approach elsewhere in the codebase.

Reply:
The problem is with the `return` statement. The `return` on an error within the `forEach` doesn't stop the request handling (although `next` is called). With a for loop, the return cancels the whole request handler

Reply:
In the cli `forEach` is fine, because an exception is thrown which also cancels the processing.